.github/workflows/conformance.yml

```diff
@@ -10,6 +10,13 @@ on:
       antrea-values:
         description: The Antrea Chart values. Multiple values can be separated with commas (e.g. key1=val1,key2=val2). Default configuration will be tested if empty.
         required: false
+      antrea-image-distro:
+        description: The Antrea image distribution to test. It could be ubuntu or ubi.
+        type: choice
+        options:
+          - ubuntu
+          - ubi
```

Comment:
I didn't know this before, so ubi image can also run on regular clusters?

Reply:
yes, it should not be a problem

Reply:
Yes. The only difference I noticed is that UBI images only have nf tables available. So it should be OK to test UBI images if the cluster is bootstrapped in nft mode

Reply:
That's a good point. Actually the iptables-wrapper has a not about this:

> RHEL / CentOS / UBI

> RHEL 7 ships iptables 1.4, which does not support nft mode. RHEL 8 ships a hacked version of iptables 1.8 that only supports nft mode. Therefore, neither can be used as a basis for a portable iptables-using container image.

I wonder if UBI 9 is different

Reply:
[9.0_release_notes](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/9.0_release_notes/index#deprecated-functionality_networking) indicates that iptables-nft has been marked as deprecated. But I think it is safe to use them in UBI 9.